autogen.sh

```diff
@@ -601,6 +601,27 @@ else
     fi
 fi
 
+########################################################################
+## Check for Python 3
+########################################################################
+
+echo_n "Checking for Python 3... "
```

Comment:
For the curious: what Python 3 features are we relying on?

Reply:
I haven't really checked. But since Python2 is deprecated there is no reason to write new scripts for Python2. By requiring Python 3, we are more future proof and have less burden of support.

Good catch for the doubled output, I'll fix it.

Reply:
> I haven't really checked. But since Python2 is deprecated there is no reason to write new scripts for Python2. By requiring Python 3, we are more future proof and have less burden of support.

Yes, but Python 2 vs. 3 on today's systems is not as straight-forward as one would like. My preference would be to make the script compatible with either. But if that is too much burden, requiring 3 is better. That would be consistent with what Yaksa already does anyway.

Reply:
> My preference would be to make the script compatible with either. But if that is too much burden,...

It is not too much burden now, but it is a burden that, once in, we are going to carry for a long time; not only for this script, but for every new scripts in order to keep the policy consistency. This is autogen, so I think it is less of an issue. But we should discuss on the general policy forward. I can easily make the revision to reflect whatever we decide then.